src/Validator/DateRangeValidator.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace AppBundle\Validator;
+
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+class DateRangeValidator extends ConstraintValidator
```

Comment:
This class would deserve some tests no ?

Reply:
Test added 👍